submissions/chri911/popup-menu/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <input type="checkbox" id="isexpanded">
+    <input type="checkbox" id="expanded">
+    <div class="container">
+        <div class="mail"><a href="#">Mail</a></div>
+        <div class="images"><a href="#">Images</a></div>
+        <div class="popup1">
+        <label for="isexpanded" id="popup"></label>
+            <div class="popup__menu">
+                <div class="popup__menu-wrapper">
+                <div class="flex">
+                <a href="#" class="item calendar">
+                    <img src="images/calendar.png" alt="">
+                    <div class="text">Calendar</div>
+                </a>
+                <a href="#" class="item mail">
+                    <img src="images/mail.png" alt="">
+                    <div class="text">Mail</div>
+                </a>
+                <a href="#" class="item news">
+                    <img src="images/news.png" alt="">
+                    <div class="text">News</div>
+                </a>
+                <a href="#" class="item map">
+                    <img src="images/maps.png" alt="">
+                    <div class="text">Map</div>
+                </a>
+                <a href="#" class="item files">
+                    <img src="images/files.png" alt="">
+                    <div class="text">Files</div>
+                </a>
+                <a href="#" class="item contacts">
+                    <img src="images/contacts.png" alt="">
+                    <div class="text">Contacts</div>
+                </a>
+                <a href="#" class="item">
+                    <img src="images/tanslation.png" alt="">
+                    <div class="text">Translate</div>
+                </a>
+                <a href="#" class="item">
+                    <img src="images/videos.png" alt="">
+                    <div class="text">Videos</div>
+                </a>
+                <a href="#" class="item">
+                    <img src="images/search.png" alt="">
+                    <div class="text">Search</div>
+                </a>
+                <a href="#" class="item photo">
+                    <img src="images/photos.png" alt="">
+                    <div class="text">Photo</div>
+                </a>
+                <a href="#" class="item apps">
+                    <img src="images/apps.png" alt="">
+                    <div class="text">Apps</div>
+                </a>
+                <a href="#" class="item photo">
+                    <img src="images/documents.png" alt="">
+                    <div class="text">Document</div>
+                </a>
+            </div>
+                <div class="more-btn1">
+                    <label for="expanded" id="more-btn">More</label>               
```

Comment:
Try to keep closer input to his label, it's easier to read and support this code in future.